<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NovyPouzDialog</class>
 <widget class="QDialog" name="NovyPouzDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nový používateľ - Lekáreň Mint</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>mint.ico</normaloff>mint.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout_13">
        <item row="0" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Pozicia</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="index">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <item>
           <property name="text">
            <string>Zákazník</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Zamestnanec</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Admin</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Login:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="login"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Heslo:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="heslo">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="zobrazHeslo">
          <property name="text">
           <string>zobraziť heslo</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Meno:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="meno"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Priezvisko:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="priezvisko"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="notAdmin">
        <property name="title">
         <string/>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QFormLayout" name="formLayout_12">
           <item row="0" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Ulica:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="ulica"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout_10">
           <item row="0" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Mesto:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="mesto"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout_11">
           <item row="0" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>PSČ:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="psc">
             <property name="maxLength">
              <number>5</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout_9">
           <item row="0" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Minuté:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDoubleSpinBox" name="minute">
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="maximum">
              <double>99999999.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout_14">
           <item row="0" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Pozícia:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="pozicia"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Vybavená objednávka:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="vybaveny">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="OkButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>index</tabstop>
  <tabstop>login</tabstop>
  <tabstop>heslo</tabstop>
  <tabstop>zobrazHeslo</tabstop>
  <tabstop>meno</tabstop>
  <tabstop>priezvisko</tabstop>
  <tabstop>ulica</tabstop>
  <tabstop>mesto</tabstop>
  <tabstop>psc</tabstop>
  <tabstop>minute</tabstop>
  <tabstop>pozicia</tabstop>
  <tabstop>vybaveny</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>OkButton</sender>
   <signal>accepted()</signal>
   <receiver>NovyPouzDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>575</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OkButton</sender>
   <signal>rejected()</signal>
   <receiver>NovyPouzDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>575</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
